### PR TITLE
tst: skip unfinished implementation tests

### PR DESCRIPTION
**Pull request by mgxd**
Preceeds #14

### CODE REVIEW

File: nitransforms/nonlinear.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def __init__(self, field, reference=None):
         self._field = field.get_data()
 
         ndim = self._field.ndim - 1
-        if self._field.shape[:-1] != ndim:
```
Comment (mgxd) [inline nitransforms/nonlinear.py]: @oesteban is this the intended behavior?

File: nitransforms/nonlinear.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def __init__(self, field, reference=None):
...
+        if len(self._field.shape[:-1]) != ndim:
```
Comment (mgxd) [inline nitransforms/nonlinear.py]: @oesteban is this the intended behavior?

Comment (oesteban) [reply nitransforms/nonlinear.py]: I'm guessing so.

Comment (oesteban) [reply nitransforms/nonlinear.py]: I'm guessing so :)

**Pull request merged**